Sprint 4/soen341-rentalapp/app/page.tsx

```diff
@@ -1,155 +1,147 @@
-import React from "react";
+'use client'
```

Comment:
dont need use client here unless needed for infinitecard thing

Reply:
I've tried removing it but it doesnt work bc one of the library uses a hook :D